pvlib/iotools/surfrad.py

```diff
@@ -146,7 +146,7 @@ def read_surfrad(filename, map_variables=True):
     metadata['surfrad_version'] = int(metadata_list[-1])
     metadata['tz'] = 'UTC'
 
-    data = pd.read_csv(file_buffer, delim_whitespace=True,
+    data = pd.read_csv(file_buffer, sep=r'\s+',
```

Comment:
The warning states:
> "FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use sep='\\s+' instead",

Is there a reason that you're using a single backslash?

Reply:
`sep='\\s+'` is equivalent to `sep=r'\s+'`, I just preferred to use an `r` string instead of escaping the backslash

Reply:
I figured - thanks for letting me know!